web-api/terraform/modules/api/pdf-generation.tf

```diff
@@ -7,8 +7,8 @@ module "pdf_generation_lambda" {
   lambda_name    = "pdf_generator_${var.environment}_${var.current_color}"
   role           = var.lambda_role_arn
   environment    = var.lambda_environment
-  timeout        = "29"
-  memory_size    = "3008"
+  timeout        = "900"
+  memory_size    = "6508"
```

Comment:
Where does this number come from?

Reply:
900 is the same timeout for async lambdas
6GB is the Memory that made this lambda capable of handling the 720 page PDF so added an extra half GB